README.md

```diff
@@ -43,6 +43,7 @@ Requires macOS 10.15 or later.
 ```bash
 sudo make install.bin
 ```
+or run `sudo make install` to install the launchd service (see below) too.
```

Comment:
This is little confusing, since it comes in the middle of the section about installing the binaries.

I think a better place would be right before the details section.

The readers are likely to follow the instructions in read order, so they will perform install.bin, and they they should continue to the install.launchd, or skip if they want to manage socket_vmnet with lima.

Maybe we need to remove the detail section and include all the info in the top level of this section.

Reply:
Fixed.

I still want to keep `<details>` as the launchd unit is not needed for Lima.